src/main/scala/geotrellis/pointcloud/spark/json/Implicits.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017 Azavea
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.pointcloud.spark.json
+
+import io.pdal.pipeline.ReaderTypes
+import geotrellis.pointcloud.vector.{Extent3D, ProjectedExtent3D}
+import geotrellis.proj4.CRS
+import geotrellis.pointcloud.util.EitherMethods
+
+import io.circe.Decoder
+
+object Implicits extends Implicits
```

Comment:
Bad rebase, json package should be deleted, all codecs from here were moved into companion objects: https://github.com/geotrellis/geotrellis-pointcloud/blob/master/src/main/scala/geotrellis/pointcloud/spark/Extent3D.scala#L29

I know that it's WIP, but to avoid bad merges and issues with non working implicits, since these codecs are incorrect and can cause issues.

Reply:
You bet.  I'll fix it.